lms/templates/dashboard.html

```diff
@@ -11,14 +11,15 @@
 from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 from openedx.core.djangolib.js_utils import dump_js_escaped_json, js_escaped_string
 from openedx.core.djangolib.markup import HTML, Text
+from adaptive_learning.config.models import AdaptiveLearningEnabledFlag
 %>
 
 <%
   cert_name_short = settings.CERT_NAME_SHORT
   cert_name_long = settings.CERT_NAME_LONG
+  adaptive_learning_enabled = AdaptiveLearningEnabledFlag.current().enabled
 %>
 
-
```

Comment:
Ok I'll take it out.
I'm a bit confused. You said to take this line out 2 days ago.
https://github.com/open-craft/edx-platform/pull/70/commits/ebf3b859dcb49734e6d58a3a98f286fdceb09465

Reply:
> I'm a bit confused. You said to take this line out 2 days ago.

Yes, because at that point [the line](https://github.com/open-craft/edx-platform/commit/ebf3b859dcb49734e6d58a3a98f286fdceb09465#diff-06f447d67e696470d5817797bfe9f7b9R25) came from a change that you made. After the updates from ecaa1eb4a9a0d7aa0e91845e20f972519919623c (in particular, the one you commented on [here](https://github.com/open-craft/edx-platform/pull/70/commits/ecaa1eb4a9a0d7aa0e91845e20f972519919623c#r149465344)), this was no longer the case. The diff just contained a "random" removal of an empty line. That's why I suggested to revert the change. (Again, skimming the entire diff after making some changes can help spot these things in advance :))